queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Find all animals whose name ends in "mon". */
+SELECT * FROM animals WHERE name LIKE '%mon%';
+/* List the name of all animals born between 2016 and 2019. */
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+/* List the name of all animals that are neutered and have less than 3 escape attempts.*/
+SELECT * FROM animals WHERE nutered = TRUE AND escape_attempts < 3;
+/* List the date of birth of all animals named either "Agumon" or "Pikachu" */
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+/* List name and escape attempts of animals that weigh more than 10.5kg */
+SELECT name, escape_attempts FROM animals WHERE weight_kg >10.5;
```

Comment:
* Kindly fix this error i get running these queries below is a screenshot HINT 💭 check the screenshot i highlighted where you spelt neutered and escape_attempt wrongly kindly make sure your spelling tallies with what you have in your schema file 👍 
![image](https://user-images.githubusercontent.com/52497246/183799069-09066644-2335-40ca-9ece-5bff6d6f083d.png)